indico/modules/rb/controllers/backend/bookings.py

```diff
@@ -308,6 +308,7 @@ def _process(self, args):
             'end_dt': args['end_dt'],
             'repeat_frequency': args['repeat_frequency'],
             'repeat_interval': args['repeat_interval'],
+            'notes': args['notes'],
```

Comment:
This should only be added if notes are enabled and the user can manage the roomt